react/ActionMenu/index.spec.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import ActionMenu, { ActionMenuItem } from './'
+
+describe('ActionMenu', () => {
+  let createRangeBackup
+  beforeAll(() => {
+    createRangeBackup = global.document.createRange
+    global.document.createRange = jest.fn(() => ({
+      setStart: () => {},
+      setEnd: () => {},
+      commonAncestorContainer: {
+        nodeName: 'BODY',
+        ownerDocument: document
+      }
+    }))
+  })
+
+  afterAll(() => {
+    global.document.createRange = createRangeBackup
+  })
+
+  it('should support null children', () => {
+    const comp = mount(
+      <ActionMenu onClose={jest.fn()}>
+        <ActionMenuItem>Item 1</ActionMenuItem>
+        null
```

Comment:
This is a null string here ?

I think you want `{ null }`.

Reply:
🤦‍♂️thanks

Reply:
Was the testing passing with the "null" string ? If yes, this could be a problem.